assets/i18n/en.json

```diff
@@ -575,7 +576,23 @@
     "I18N_TOPNAV_SIGN_IN_WITH_GOOGLE": "Sign in with Google",
     "I18N_TOPNAV_TEACH_WITH_OPPIA": "Teach with Oppia",
     "I18N_TOPNAV_TOPICS_AND_SKILLS_DASHBOARD": "Topics and Skills Dashboard",
+    "I18N_TOPNAV_WELCOME_GUIDE_PAGE": "Welcome Guide",
     "I18N_TOTAL_SUBSCRIBERS_TEXT": "You have a total of <[totalSubscribers]> subscribers.",
     "I18N_UNSUBSCRIBE_BUTTON_TEXT": "Unsubscribe",
+    "I18N_WELCOME_GUIDE_TITLE": "Teachers and Parents: A Guide to Oppia",
+    "I18N_WELCOME_GUIDE_BROWSER-BULLET_1": "Search bar used to search for a specific exploration",
+    "I18N_WELCOME_GUIDE_BROWSER-BULLET_2": "Category Filter to filter explorations by topic (music, math, biology, etc)",
```

Comment:
Please confirm this, i dont think ```math``` is a word, it's ```maths``` or ```mathematics```

Reply:
Done!